### PR TITLE
DOC: add hardware requirements

### PR DESCRIPTION
**Pull request by wagner-certat**

### CODE REVIEW

File: docs/user/hardware-requirements.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- 10 GB disk size
+
+**Depending on your data input, you will need the twentiethfold of the input data size as memory for processing.**
+
+When using `Redis persistence <http://redis.io/topics/persistence>`_, you will additionally need twice as much memory for Redis.
+
+Disk space
+==========
+
+Disk space is only relevant if you save your data to a file, whic is not recommended for production setups, and only useful for testing and evaluation.
```
Comment (pp-) [inline docs/user/hardware-requirements.rst:38]:
```suggestion
Disk space is only relevant if you save your data to a file, which is not recommended for production setups, and only useful for testing and evaluation.
```

**Comment by wagner-certat** [comment]
Note to myself: Afterwards add it to https://github.com/melicertes/docs/

File: docs/user/hardware-requirements.rst (hunk 1 above)
Comment (wagner-certat) [reply docs/user/hardware-requirements.rst]: Thanks, fixed

**Review by schacht-certat — APPROVED** [review]

**Review by waldbauer-certat — APPROVED** [review]

**Comment by wagner-certat** [comment]
Merged in https://github.com/certtools/intelmq/commit/2763398c9e756fa984be9d6cd8c5ef6435eeedb6

**Pull request closed without merging**